src/modal.css

```diff
@@ -0,0 +1,114 @@
+.filters-container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+  button {
+    display: inline-block;
+    background-color: #262626;
+    color: white;
+    width: fit-content;
+    height: 42px;
+    padding: 8px 16px;
+    border-radius: 21px;
+    font-size: 19px;
+    line-height: 19px;
+    font-weight: 700;
+    border: 1px solid #262626;
+  }
+  .disabled-filter {
+    background-color: white;
+    color: #262626;
+    border: 1px solid #dbd9d7;
+  }
+}
+
+.modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.67);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.modal-content {
+  background: white;
+  padding: 40px 24px;
+  border-radius: 8px;
+  width: 496px;
+  text-align: center;
+  margin: 16px;
+}
+
+.search-bar input[type="text"] {
+  width: 100%;
+  height: 50px;
+  box-sizing: border-box;
+  margin-top: 16px;
+  padding: 12px 16px;
+  font-size: 14px;
+  border: 1px solid #dbd9d7;
+  border-radius: 4px;
+}
+
+form button {
+  display: inline-block;
+  background-color: #262626;
+  color: white;
+  width: fit-content;
+  font-size: 19px;
+  font-weight: 700;
+  padding: 8px 16px;
+}
+
+.search-buttons {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.close-button {
+  background-color: #5d7074;
+  border: none;
+}
+
+.submit-button {
+  background-color: #303d3f;
+  border: none;
+}
+
+.title-and-button .material-symbols-outlined {
+  font-family: "Material Symbols Outlined";
+  font-size: 16px;
+  color: #262626;
+  border: 1px solid #dbd9d7;
+  border-radius: 50%;
+  padding: 8px;
+  position: relative;
```

Comment:
You're rather styling the button here than the icon.
So, instead of using `.material-symbols-outlined` as a selector, `button` or a new dedicated class for that specific button would be a better selector.